install-dev/data/xml/configuration.xml

```diff
@@ -387,7 +387,7 @@
     <value>0</value>
   </configuration>
   <configuration id="PS_IMAGE_QUALITY" name="PS_IMAGE_QUALITY">
-    <value>jpg</value>
+    <value>1</value>
```

Comment:
What is the reason for putting 1 as a default value here ?  (I'm not sure I understand, maybe I forgot something?)

Reply:
It's a BC Break, I think you should keep `jpg` there.